vavr/src-gen/main/java/io/vavr/API.java

```diff
@@ -5123,7 +5123,7 @@ public static final class Case0<T, R> implements Case<T, R> {
             private static final long serialVersionUID = 1L;
 
             private final Pattern0<T> pattern;
-            private final Function<? super T, ? extends R> f;
+            private final transient Function<? super T, ? extends R> f;
```

Comment:
API is a generated class hence this change should be done in Generator class. I have taken care of this in my PR: https://github.com/vavr-io/vavr/pull/2880/files#diff-3b19b7314b0707a187940dc9e7f5bf8aa42e99ff0804aedae5e36eb9c764be3aR1107